<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTreeView" name="treeView">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>310</width>
      <height>400</height>
     </rect>
    </property>
   </widget>
   <widget class="QTreeView" name="treeView_2">
    <property name="geometry">
     <rect>
      <x>325</x>
      <y>5</y>
      <width>500</width>
      <height>400</height>
     </rect>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="AddParticipant">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>410</y>
      <width>130</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajouter un participant</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ResetAssociations">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>410</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Reinitialiser les associations</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ComputeAssociations">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>410</y>
      <width>165</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculer la meilleur combinaison</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DeleteParticipant">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>410</y>
      <width>130</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Supprimer le participant</string>
    </property>
   </widget>
  </widget>
  <action name="actionNouveau">
   <property name="text">
    <string>Nouveau</string>
   </property>
  </action>
  <action name="actionOuvrir">
   <property name="text">
    <string>Ouvrir</string>
   </property>
  </action>
  <action name="actionSauvegarder">
   <property name="text">
    <string>Sauvegarder</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
